docs/pom.xml

```diff
@@ -55,15 +55,15 @@
     </licenses>
 
     <scm>
-      <connection>scm:svn:http://svn.apache.org/repos/asf/incubator/ambari</connection>
+      <connection>scm:svn:https://svn.apache.org/repos/asf/incubator/ambari</connection>
```

Comment:
I believe urls here and below are outdated. We should point to GIT repo
https://svn.apache.org/repos/asf/ambari/branches/